sky/packages/sky/lib/src/rendering/statistics_box.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:sky/src/rendering/box.dart';
+import 'package:sky/src/rendering/object.dart';
+
+class StatisticsBox extends RenderBox {
+
```

Comment:
nit nit: if you have a blank line at the top you should have one at the bottom (or vice versa, no blank line either place)